apax/cli/templates/train_config_full.yaml

```diff
@@ -44,6 +43,8 @@ model:
     r_max: 6.0
     r_min: 0.5
 
+  ensemble: null
+
```

Comment:
Just for my future training runs: `n_models` has been renamed `ensemble` but is still `int|None`?

Reply:
no, ensemble lets' you choose the kind of ensemble, hoe many members and in the case of shallow ensembles, whether or not to compute force variance.